.gitignore

```diff
@@ -10,3 +10,4 @@ venv*/
 .python-version
 build/
 dist/
+.idea/
```

Comment:
Tip: you can put this in a [global `~/.gitignore`](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html). :-) 

```suggestion
```